RELEASE_NOTES.md

```diff
@@ -1,5 +1,76 @@
+* Fix error thrown when randomly generated private key starts with a zero byte which would cause message signing failures. Transaction signing is not affected.
+* Fix bug which caused only the first consolidation transaction to be returned from `consolidateTransaction()` for v1 wallets instead of all transations.
+
+### Other changes
+* Updated the install instructions for BitGoJS to `npm install bitgo` instead of cloning the project directly. This has an effect on how to run `bitgo-express`. To install and run `bitgo-express`, the recommended command is `npm install -g bitgo && npm explore -g bitgo -- bin/bitgo-express`.
+* Upgraded eslint to 5.15.1, which entails dropping support for development on BitGoJS on node versions below 6.14.0. If you need to develop on node 6.x, please use at least 6.14.0, and consider upgrading soon as [node 6 is scheduled to reach end-of-life on April 30, 2019](https://github.com/nodejs/Release#release-schedule). Only users of BitGoJS who are contributing source code changes are affected by this requirement. End users can continue using node versions >=6.12.3, but please upgrade soon.
+* Remove karma browser testing framework. We will be revamping our browser testing in a future release.
+
+### Common Issues when Upgrading
+
+#### Warning on installation
+
+You may notice a warning when installing BitGoJS about using a deprecated script type:
+```
+npm WARN prepublish-on-install As of npm@5, `prepublish` scripts are deprecated.
+npm WARN prepublish-on-install Use `prepare` for build steps and `prepublishOnly` for upload-only.
+npm WARN prepublish-on-install See the deprecation note in `npm help scripts` for more information.
+```
+
+This is expected, and we cannot yet change to using a prepare script because this script type is not yet available in some of our supported npm versions.
+
+#### Potential error when starting `bitgo-express`
+If you see the following error when running `bin/bitgo-express`, it means the typescript files have not been compiled. 
+```
+module.js:478
+    throw err;
+    ^
+
+Error: Cannot find module '../dist/src/expressApp'
+    at Function.Module._resolveFilename (module.js:476:15)
+    at Function.Module._load (module.js:424:25)
+    at Module.require (module.js:504:17)
+    at require (internal/module.js:20:19)
+    at Object.<anonymous> (/bitgo-dep/node_modules/bitgo/bin/bitgo-express:5:66)
+    at Module._compile (module.js:577:32)
+    at Object.Module._extensions..js (module.js:586:10)
+    at Module.load (module.js:494:32)
+    at tryModuleLoad (module.js:453:12)
+    at Function.Module._load (module.js:445:3)
+    at Module.runMain (module.js:611:10)
+    at run (bootstrap_node.js:394:7)
+    at startup (bootstrap_node.js:160:9)
+    at bootstrap_node.js:507:3
+```
+To fix this, You can compile the typescript source manually by running `npm explore bitgo -- npm run prepublish`.
```

Comment:
getting the following error but maybe i'm not doing this correctly? 
<img width="523" alt="Screen Shot 2019-04-09 at 10 08 13 AM" src="https://user-images.githubusercontent.com/43550949/55820218-73cb7100-5aaf-11e9-86b2-a8a761e17af7.png">

Reply:
Version 5 hasn't been published yet.